gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Channelizer.java

```diff
@@ -87,6 +88,7 @@ default String getScheme(final boolean sslEnabled) {
     abstract class AbstractChannelizer extends ChannelInitializer<SocketChannel> implements Channelizer {
         protected Connection connection;
         protected Cluster cluster;
+        protected SslHandler sslHandler;
```

Comment:
Is this safe to do? The SslHandler isn't annotated with "Shareable".

Reply:
There should not be multiple threads setting the SslHandler as it is initialized as part of the Connection constructor which calls `new Channelizer.HttpChannelizer()` and eventually `ChannelInitializer.initChannel` which only executes `initChannel(Channel)` once per `ChannelHandlerContext`.